lib/components/array/Filter.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 
 const render = ({ children, filter, items }) =>
-  items ? children(items.filter(filter)) : null;
+  items ? <>{children(items.filter(filter))}</> : null;
```

Comment:
was it breaking without the React Fragment wrapper, it shouldn't be? I don't think prettier supports that syntax yet, we have to use <React.Fragment> for now.